### PR TITLE
Removed OSGB.py and modified unit tests that use OSGB to use standard…

### PR DESCRIPTION
**Pull request by PaulAbernethy**
… UKVX grid

Reference issue if exists

Testing:
 - [x ] Ran tests and they passed OK

### CODE REVIEW

File: lib/improver/tests/wind_downscaling/test_roughnesscorrection.py [hunk 1]
```diff
@@ -37,16 +37,60 @@
...
 from iris.tests import IrisTest
+from iris.coord_systems import (LambertAzimuthalEqualArea,
+                                GeogCS)
 import numpy as np
-
-from improver.grids.osgb import OSGBGRID
 from improver.constants import RMDI
 from improver.wind_downscaling import RoughnessCorrection
 
+ELLIPSOID = GeogCS(semi_major_axis=6378137.0,
```
Comment (gavinevans) [inline lib/improver/tests/wind_downscaling/test_roughnesscorrection.py]: Should the definitions of the `ELLIPSOID` and `STANDARD_GRID_CCRS` already in https://github.com/metoppv/improver/blob/1e7827af4e8d2acf7dec9e0dc4ffb8ab485aebe3/lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py#L106 be factored out, so we don't keep redefining this grid?

File: lib/improver/tests/wind_downscaling/test_roughnesscorrection.py [hunk 2]
```diff
@@ -37,16 +37,60 @@
 import iris
 from iris.coords import AuxCoord
 from iris.tests import IrisTest
+from iris.coord_systems import (LambertAzimuthalEqualArea,
```
Comment (wxtim) [inline lib/improver/tests/wind_downscaling/test_roughnesscorrection.py]: pylint returns an error for this: 
```
E: 40, 0: No name 'LambertAzimuthalEqualArea' in module 'iris.coord_systems' (no-name-in-module)
```
I've no idea why though

**Comment by wxtim** [comment]
Apart from the comment from Gavin, and the one I added I can't see any issues with this. I've run the unit tests manually. If pylint were correct then this ought not work.

File: lib/improver/tests/wind_downscaling/test_roughnesscorrection.py (hunk 1 above)
Comment (PaulAbernethy) [reply lib/improver/tests/wind_downscaling/test_roughnesscorrection.py]: We could do that, sure. Longer term, will this be the best place to define it? It seems a bit hidden away in a unit test script when maybe it should be in a constants file or something.

File: lib/improver/tests/wind_downscaling/test_roughnesscorrection.py (hunk 2 above)
Comment (PaulAbernethy) [reply lib/improver/tests/wind_downscaling/test_roughnesscorrection.py]: It makes the same complaint in improver/lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py where it was initially defined. I have no idea why it complains either.

**Comment by PaulAbernethy** [comment]
After a discussion with Gavin, we decided that it might be sensible to have a new top-level module called grids.py that stores the standard grid information, rather than it being defined in a random unit test. I have updated the unit tests that defined the STANDARD_GRID_CRSS and ELLIPSOID parameters to import these values from this new module instead.

We also figured out why pylint was complaining: This is due to the site.init file pointing to an old version of pylint. Removing the line 'export PYLINT=/usr/local/sci/bin/pylint' fixes this issue.

**Comment by gavinevans** [comment]
Rebased following the merger of #360.

**Pull request merged**